### PR TITLE
FP8 TN Compute for Grouped GEMMs

### PR DESCRIPTION
**Pull request by ssuyuanchang**
1. Add yaml to FreeSize
2. Add gtest case to verify

### CODE REVIEW

**Review by imcarsonliao — COMMENTED** [review]
please ensure the gtest has been added and validated

File: clients/gtest/matmul_gtest.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@ Definitions:
     - { transA: N, transB: T }
     - { transA: T, transB: N }
     - { transA: T, transB: T }
+    
+  - &transA_transB_tn
```
Comment (jichangjichang) [inline clients/gtest/matmul_gtest.yaml]: don't  create new set for only one case

File: clients/gtest/matmul_gtest.yaml [hunk 2]
```diff
@@ -1014,6 +1017,21 @@ Tests:
...
   algo_method: [0, 2]
 
+- name: matmul_groupedgemm_f8
+  category: pre_checkin
+  function:
+    matmul: *f8_precision_dst_fp16
+  M: [127, 129]
+  N: [127, 129]
+  K: [127, 129]
+  transA_transB: *transA_transB_tn
```
Comment (jichangjichang) [inline clients/gtest/matmul_gtest.yaml]: transA: T
  transB: N

Comment (ssuyuanchang) [reply clients/gtest/matmul_gtest.yaml]: done

**Comment by ssuyuanchang** [comment]
> please ensure the gtest has been added and validated

Already added in gtest

**Comment by ssuyuanchang** [comment]
[33m=========== [32m78 passed[0m, [33m[1m32 skipped[0m, [33m[1m5 warnings[0m[33m in 8706.45s (2:25:06)[0m[33m ============[0m
py310: exit 0 (8706.62 seconds) /josh/hipBLASLt/tensilelite> py.test -v --basetemp=/josh/hipBLASLt/tensilelite/.tox/py310/tmp --junit-xml=/josh/hipBLASLt/tensilelite/python_tests.xml --junit-prefix=py310 --color=yes -n 4 --prebuilt-client=/josh/hipBLASLt/tensilelite/.tox/py310/client/0_Build/client/tensile_client Tensile/Tests -m common --capture=sys pid=223002
  py310: OK (8837.46=setup[11.09]+cmd[0.85,118.90,8706.62] seconds)
  congratulations :) (8837.53 seconds)

**Review by AndySu12 — APPROVED** [review]

**Comment by jichangjichang** [comment]
please file the result of hipblaslt-test

**Comment by ssuyuanchang** [comment]
Note: Google Test filter = \*matmul_groupedgemm_f8\*
[==========] Running 2 tests from 1 test suite.
[----------] Global test environment set-up.
[----------] 2 tests from _/matmul_test
[ RUN      ] _/matmul_test.matmul/pre_checkin_matmul_groupedgemm_f8_f8_rf8_rf16_rf16_rf32_r_TN_127_127_127_1_127_127_0_127_127_1_GG2_APIExt_APIAlgoIndex
[       OK ] _/matmul_test.matmul/pre_checkin_matmul_groupedgemm_f8_f8_rf8_rf16_rf16_rf32_r_TN_127_127_127_1_127_127_0_127_127_1_GG2_APIExt_APIAlgoIndex (1478 ms)
[ RUN      ] _/matmul_test.matmul/pre_checkin_matmul_groupedgemm_f8_f8_rf8_rf16_rf16_rf32_r_TN_127_127_127_1_127_127_2_127_127_1_GG2_APIExt_APIAlgoIndex
[       OK ] _/matmul_test.matmul/pre_checkin_matmul_groupedgemm_f8_f8_rf8_rf16_rf16_rf32_r_TN_127_127_127_1_127_127_2_127_127_1_GG2_APIExt_APIAlgoIndex (70 ms)
[----------] 2 tests from _/matmul_test (1551 ms total)

[----------] Global test environment tear-down
[==========] 2 tests from 1 test suite ran. (1587 ms total)
[  PASSED  ] 2 tests.
hipBLASLt version: 1100

command line: ./build_tn_new/release/clients/staging/hipblaslt-test --gtest_filter=\*matmul_groupedgemm_f8\*

**Review by jichangjichang — APPROVED** [review]

**Pull request merged**